frontend/src/pages/ActivityReport/Pages/supportingAttachments.js

```diff
@@ -22,7 +22,7 @@ const SupportingAttachments = ({
   return (
     <>
       <Helmet>
-        <title>Supporting attachments</title>
+        <title>Supporting Attachments</title>
```

Comment:
we don't want the personalization in the individual activity report pages? report ID, or some other distinguishing factor?